### PR TITLE
Add "skip_missing" option to podman_image_info module

### LINKED ISSUE #57899: Ansible podman_image_info module doesn't have to fail if images doesn't exist
##### SUMMARY
When running podman_image_info on list of images and one of them doens't exist, the module fails. It's not always a desired behavior. Sometimes we might want just to get data about existing images and continue, like it's done for example in docker_image_info module.
We need a special option for that.

##### ISSUE TYPE
- Feature Idea

##### COMPONENT NAME
podman_image_info module

##### ADDITIONAL INFORMATION
Example:
```yaml
- name: Gather info on several images and don't fail if some missing
  podman_image_info:
    name:
      - redis
      - idontexist
    skip_missing: true
```

### PR DESCRIPTION
**Pull request by sshnaidm**
##### SUMMARY

Fixes #57899
If some of images don't exist we need an option not to fails task
but just to collect info about all existing images. It mimics
logic of docker_image_info module as well, which don't fail if
an image doens't exist.
Make this optional and set to false by default to keep backward
compatibility.
##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
podman_image_info

##### ADDITIONAL INFORMATION

Test: `podman pull nginx`
then run:
```yaml
- name: Gather info on several images and don't fail if some missing
  podman_image_info:
    name:
      - nginx
      - idontexist
    skip_missing: true
```
and it shouldn't fail, but give info about nginx image only.

### CODE REVIEW

File: lib/ansible/modules/cloud/podman/podman_image_info.py [hunk 1]
```diff
@@ -31,7 +31,10 @@
   name:
     description:
       - List of tags or UID to gather info about. If no name is given return info about all images.
-
+  skip_missing:
+    version_added: 2.9
+    description:
+      - If image doesn't exist don't fail the task, just inlcude existing only in output.
```
Comment (ssbarnea) [inline lib/ansible/modules/cloud/podman/podman_image_info.py]: typo on include

File: lib/ansible/modules/cloud/podman/podman_image_info.py [hunk 2]
```diff
@@ -47,6 +50,13 @@
     name:
       - redis
       - quay.io/bitnami/wildfly
+
+- name: Gather info on several images and don't fail if some missing
```
Comment (ssbarnea) [inline lib/ansible/modules/cloud/podman/podman_image_info.py]: "avoid" sounds better than "don't"

**Review by ssbarnea — CHANGES_REQUESTED** [review]
two spelling issues

File: lib/ansible/modules/cloud/podman/podman_image_info.py (hunk 1 above)
Comment (sshnaidm) [reply lib/ansible/modules/cloud/podman/podman_image_info.py]: fixed

File: lib/ansible/modules/cloud/podman/podman_image_info.py (hunk 2 above)
Comment (sshnaidm) [reply lib/ansible/modules/cloud/podman/podman_image_info.py]: fixed

**Pull request closed without merging**